src/shared/components/MarkDownEditor/extensions/Extensions.hooks.ts

```diff
@@ -67,7 +67,7 @@ export const useUploadMethods = ({
     if (!inputRef.current?.files?.length) return;
 
     const file = inputRef.current.files[0];
-    if (file.size > fileSizeExceeded) {
+    if (file.size >= fileSizeExceeded) {
```

Comment:
could you please add a couple of tests to cover this condition?

Reply:
There are no tests for this hook. A corresponding task has been created: 
https://mindlogger.atlassian.net/browse/M2-5740.